lib/rialto/etl/logging.rb

```diff
@@ -12,9 +12,11 @@ module Logging
       def logger
         @logger ||= Yell::Logger.new(:null).tap do |logger|
           # Everything to stderr
-          logger.adapter :stderr
+          # logger.adapter :stderr
+          logger.adapter :datefile, 'rialto_etl.log', level: %i[debug info warn]
```

Comment:
We use `:warn` all over the place. :+1: to either using `:error` in all the spots where we actually want to know that happened, or including `:warn` in the error log.